members/D000622.yaml

```diff
@@ -2,83 +2,91 @@ bioguide: D000622
 contact_form:
   method: post
   action: ""
+  usePhantomLogging: true
   steps:
-    - visit: "https://www.duckworth.senate.gov/content/contact-senator"
+    - visit: "https://www.duckworth.senate.gov/connect/email-tammy"
     - find:
-        - selector: form#webform-client-form-78 input#edit-submitted-constituent-information-first-name
+        - selector: div#main_column .feedback-block.opinion a
```

Comment:
you probably don't need this find if you're going to click on it next. the click_on step will try to find it first before clicking

Reply:
Oh got it. I was following a code example I found in the repo. I can delete the -find and just leave -click_on. Would that work?

Reply:
yeah i think so. you should test it to make sure that it still works without the find